src/main/resources/css/reporting.css

```diff
@@ -50,8 +50,12 @@ a:hover {
 }
 
 /* left line along each section */
+.element, .steps, .hooks-after, .hooks-before {
+    box-shadow: -1px 0px lightgray;
```

Comment:
I understand that the black color was not very stylish however now the color remains the same - can you change to color to a little more darker is once you hover the section you see when it starts and ends?  some compromise between black and lightgray :)